### DISCUSSION TITLE
rTorrent & screen

### DISCUSSION THREAD

**Discussion by vdias** [swizzin/swizzin#710] [General] [answered]
Why you guys keep using screen with rtorrent, if rtorrent already have a functional daemon module available???

**Comment by flying-sausages** [comment, marked as answer]
Because a lot of people want to get into the actual CLI interface from time to time. AFAIK it is not possible to access a daemonised session over the CLI in any way at all. It would also complicate our deployment because we would have to handle different cases for different versions, because IIRC it's only 0.9.7+ that supports it.

With that said, you could easily make a tiny config to the rc files and the service file, and run with that.
